handbook/engineering/search/index.md

```diff
@@ -67,9 +67,9 @@ _Updated 2020-09-11_
       - Unblock the ability to add responsive and fast-loading search results.
       - Return results faster for large result sets, e.g. indexed repositories in very large codebases.
   - **Plan:** Streaming search
-  - **Owner:** Keegan
-  - **Status:** [In progress](perf.md)
-  - **Estimated completion:** 3.21
+  - **Owners:** Keegan, Juliana
+  - **Status:** In progress
+  - **Estimated completion:** 3.22
```

Comment:
Given what we discussed yesterday, I feel like the work to fully complete this goal (streaming search is rolled out by default for all users & all result types) is a bit much to achieve before 3.21 branch cut (2020-10-14). I bumped the estimated completion for this to 3.22. Wdyt?

Reply:
agreed